torch/onnx/_patch_torch.py

```diff
@@ -8,13 +8,15 @@
 from torch._C import _onnx as _C_onnx
 from torch.onnx import _deprecation
 from torch.onnx._globals import GLOBALS
+from torch.onnx._internal import _beartype
 
 
 # TODO(#78694): Refactor the patching process to make it more transparent to users.
+@_beartype.beartype
 def _graph_op(
     g: torch._C.Graph,
     opname: str,
-    *raw_args: torch._C.Value,
+    *raw_args: Union[torch.Tensor, torch._C.Value],
```

Comment:
Is this right? Can raw_args be torch.Tensor? (They exist in tests)

Reply:
I would say no, but could be wrong.
Which test has it? let's look into it

Reply:
Can we clean up and remove it (preferably in a separate PR)? I guess Meta internal test is likely to scream though :(

Reply:
Sure thing!

Reply:
Letting a non Value goes true the most generic function for symbolics open the door for many bugs. With this, any symbolic can wrongly pass `torch.Tensor` when it was not supposed to. IMO we need to fix this as opposed to relax the type checking to allow CI to pass. Hardening the code is always best, even if it needs more refactoring

Reply:
In a follow-up PR. This is already too big